<!DOCTYPE html>
<html lang="en">
<head>
  
  
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    
    
    <link rel="shortcut icon" href="../img/favicon.ico">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
    <title>Analysis of Post-Translational Modifications using CCMS tools - MassIVE Documentation</title>
    <link href="../css/bootstrap-3.3.7.min.css" rel="stylesheet">
    <link href="../css/font-awesome-4.7.0.css" rel="stylesheet">
    <link href="../css/base.css" rel="stylesheet">
    <link rel="stylesheet" href="../css/highlight.css">
    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
    <![endif]-->

    <script src="../js/jquery-3.2.1.min.js"></script>
    <script src="../js/bootstrap-3.3.7.min.js"></script>
    <script src="../js/highlight.pack.js"></script>
    
    <base target="_top">
    <script>
      var base_url = '..';
      var is_top_frame = false;
        
        var pageToc = [
          {title: "Analysis of Post-Translational Modifications using CCMS tools", url: "#_top", children: [
              {title: "Analyzing known modifications", url: "#analyzing-known-modifications" },
              {title: "Analyzing peptide variants and peptidoforms", url: "#analyzing-peptide-variants-and-peptidoforms" },
              {title: "Analyzing hypermodified regions", url: "#analyzing-hypermodified-regions" },
              {title: "Analyzing putative novel modifications", url: "#analyzing-putative-novel-modifications" },
          ]},
        ];

    </script>
    <script src="../js/base.js"></script> 
</head>

<body>
<script>
if (is_top_frame) { $('body').addClass('wm-top-page'); }
</script>



<div class="container-fluid wm-page-content">
  <a name="_top"></a>
    

    

    <h1 id="analysis-of-post-translational-modifications-using-ccms-tools">Analysis of Post-Translational Modifications using CCMS tools</h1>
<p>We use two different workflows for identification of modifications:</p>
<ul>
<li>
<p><a href="../workflows/maestro/">Maestro workflow</a>: identification of peptide-spectrum matches (PSMs) by i) spectral library search (MSPLIT) for detection of known reference peptides, followed by ii) restricted database search with known modifications (MSGF+), and finally followed by iii) open-modification search (MODa) for identification of peptides with unexpected modifications.</p>
<ul>
<li>Maestro primarily identifies modified <strong>peptide variants</strong>: distinct pairs (S,M) where S is a peptide sequence and M is the summed mass of all 'delta-mass' modifications assigned on sequence S. This abstraction is required for proper control of false discovery rates at PSM and peptide levels in Maestro; the resolution of delta-masses into putative modifications and modification sites is done with ModDecode as described below.</li>
</ul>
</li>
<li>
<p><a href="../workflows/moddecode/">ModDecode workflow</a>: post-processing of modified PSMs for i) resolution of delta-masses into known UniMod modifications for PSMs with unexpected modifications, ii) site assignment at fixed false localization rate and iii) calculation of AlignGF p-values between differently-modified versions of the same peptide.</p>
<ul>
<li>ModDecode primarily identifies modified <strong>peptidoforms</strong>: peptide sequences with site-resolved modifications whose masses were matched to known UniMod entries for putative resolution into (possibly combinations of) known modifications or amino acid polymorphisms.</li>
</ul>
</li>
</ul>
<hr />
<p>The following sections focus on different aspects of the analysis of modifications and modified peptides. All links below use two example jobs whose results will be reported soon: a <a href="https://proteomics2.ucsd.edu/ProteoSAFe/status.jsp?task=de809a3717cc4d96a3a5257e290351d5">Maestro job</a> co-analyzing colon samples from the draft proteomes and a colon cancer dataset and the corresponding <a href="https://proteomics2.ucsd.edu/ProteoSAFe/status.jsp?task=790e82c2ef1541748521db1e94b24fe0">ModDecode</a> job with resolved modifications and peptidoforms.</p>
<h3 id="analyzing-known-modifications">Analyzing known modifications</h3>
<ol>
<li>Start with ModDecode list of <a href="https://proteomics2.ucsd.edu/ProteoSAFe/result.jsp?task=790e82c2ef1541748521db1e94b24fe0&amp;view=t_group_by_knownmod">UniMod modifications</a>; the default sorting order is by decreasing number of distinct peptides identified with each modification (<strong>"#Pepts"</strong>). </li>
<li>Additional important criteria for inspection of modifications:<ul>
<li><strong>"% Explained"</strong> &gt;= 0.5: filter for modifications with at least one PSM with &gt;= 50% intensity explained by theoretical ions derived from the modified peptidoform sequence</li>
<li><strong>"MaxModFrags"</strong> &gt;= 2: filter for modifications with at least one PSM with &gt;= 2 spectrum peaks assigned to modified ion fragments (exclude this filter for labile modifications)</li>
<li><strong>"PValue"</strong> &gt;= 10: filter for modifications with at least one PSM matching a differently-modified (e.g., unmodified) PSM of the same peptide with an AlignGF p-value &lt;= 1e-10</li>
</ul>
</li>
</ol>
<p>Other considerations for the inspection of modifications</p>
<ul>
<li>The identification of a modification in a sample is only as good as the best PSM evidence that can be provided for that modification. See below for considerations for identification of modified peptidoforms.</li>
<li>Different modifications can have very similar mass; for example, several modifications and amino acid polymorphisms have <a href="https://proteomics2.ucsd.edu/ProteoSAFe/result.jsp?task=790e82c2ef1541748521db1e94b24fe0&amp;view=t_group_by_knownmod#%7B%22main.mass_lowerinput%22%3A%2214%22%2C%22main.mass_upperinput%22%3A%2214%22%7D">integer mass of 14 Da</a> and very close or indistinguishable mono-isotopic masses.</li>
<li>The same modification can occur on multiple different amino acids and sites: for example, the first 3 modified peptidoforms for peptide VGAHAGEYGAEALER <a href="https://proteomics2.ucsd.edu/ProteoSAFe/result.jsp?task=790e82c2ef1541748521db1e94b24fe0&amp;view=t_group_by_spectrum#%7B%22modFragNum_lowerinput%22%3A%222%22%2C%22calcPV_lowerinput%22%3A%2210%22%2C%22minRatio_lowerinput%22%3A%22.5%22%2C%22table_sort_history%22%3A%22curatedPept_asc%3BunmodPept_asc%3BminRatio_dsc%22%2C%22numMod_upperinput%22%3A%221%22%2C%22unmodPept_input%22%3A%22.VGAHAGEYGAEALER.%22%2C%22ModAnnotation_input%22%3A%22%2B16%2C%22%7D">shown on this list</a> localize the +15.995 Da mass offset to 3 different sites (once as an oxidation on Y and twice as polymorphisms of A to S, all resulting from the exact same change in elemental composition), each supported by multiple site-localizing b and y ions at mass error &lt;= 10 ppm.</li>
</ul>
<h3 id="analyzing-peptide-variants-and-peptidoforms">Analyzing peptide variants and peptidoforms</h3>
<p>Analyzing peptide variants
* First establish that it's the correct peptide - inspect the fragmentation pattern and compare it to that of related (possibly unmodified) variants of the same peptide
* (1) Copy input PSM string from ModDecode row and use it to filter Clusters view in Maestro and (2) click on the "Spectral networks" link to open the network
* (1a) Copy input PSM string from ModDecode row and use it to filter Spectral Networks view in Maestro + Click "Visualize Network" and (2) Label nodes with peptide strings and (3) Search for peptide string</p>
<p>Analyzing peptide modification masses
* Are there are other, higher-reliability modifications of similar mass identified on the same peptide? If yes, is there enough evidence to call an additional modification?
   - Example peptide <a href="https://proteomics2.ucsd.edu/ProteoSAFe/result.jsp?task=790e82c2ef1541748521db1e94b24fe0&amp;view=t_group_by_spectrum#%7B%22Index_lowerinput%22%3A%221337854%22%2C%22Index_upperinput%22%3A%221337854%22%7D">ATAASSSSLE+13.979K</a>, with Carbonyl modification on E10; note mass of highest-intensity modified y-ion ion y7 at 751.35, matching the theoretical mass 751.347 with mass error &lt; 5 ppm. Other modified y ion masses further confirm this assignment.
   - Example peptide <a href="https://proteomics2.ucsd.edu/ProteoSAFe/result.jsp?task=790e82c2ef1541748521db1e94b24fe0&amp;view=t_group_by_spectrum#%7B%22Index_lowerinput%22%3A%221660252%22%2C%22Index_upperinput%22%3A%221660252%22%7D">ATAASSSSLE+14.016K</a>, with Methylation modification on E10; note mass of highest-intensity modified y-ion ion y7 at 751.387, matching the theoretical mass 751.387 with mass error &lt; 5 ppm. Other modified y ion masses further confirm this assignment.</p>
<p>Analyzing modification sites
* Is the same modification (or a different modification of the same mass) identified elsewhere on the same peptide? If yes, are there enough site-assignment peaks between the two sites to call an additional modification site?
* Filter the PSMs view for <strong>"DtFlrPeaks"</strong> &gt;= 2</p>
<h3 id="analyzing-hypermodified-regions">Analyzing hypermodified regions</h3>
<h3 id="analyzing-putative-novel-modifications">Analyzing putative novel modifications</h3>

  <br>
    

    <br>
</div>

<footer class="container-fluid wm-page-content">
  <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a> using <a href="https://github.com/gristlabs/mkdocs-windmill">Windmill</a> theme by Grist Labs.</p>
</footer>

</body>
</html>