<!DOCTYPE html>
<html lang="en">
<head>
  
  
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    
    
    <link rel="shortcut icon" href="../img/favicon.ico">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
    <title>Reanalyze Spectra - MassIVE Documentation</title>
    <link href="../css/bootstrap-3.3.7.min.css" rel="stylesheet">
    <link href="../css/font-awesome-4.7.0.css" rel="stylesheet">
    <link href="../css/base.css" rel="stylesheet">
    <link rel="stylesheet" href="../css/highlight.css">
    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
    <![endif]-->

    <script src="../js/jquery-3.2.1.min.js"></script>
    <script src="../js/bootstrap-3.3.7.min.js"></script>
    <script src="../js/highlight.pack.js"></script>
    
    <base target="_top">
    <script>
      var base_url = '..';
      var is_top_frame = false;
        
        var pageToc = [
        ];

    </script>
    <script src="../js/base.js"></script>
    <script>
        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

        ga('create', 'UA-8412213-5', 'ccms-ucsd.github.io');
        ga('send', 'pageview');
    </script> 
</head>

<body>
<script>
if (is_top_frame) { $('body').addClass('wm-top-page'); }
</script>



<div class="container-fluid wm-page-content">
  <a name="_top"></a>
    

    
    
      
    

  <div class="row wm-article-nav-buttons" role="navigation" aria-label="navigation">
    
    <div class="wm-article-nav pull-right">
      <a href="../access_public_datasets/" class="btn btn-xs btn-default pull-right">
        Next
        <i class="fa fa-chevron-right" aria-hidden="true"></i>
      </a>
      <a href="../access_public_datasets/" class="btn btn-xs btn-link">
        Access Public Datasets
      </a>
    </div>
    
    <div class="wm-article-nav">
      <a href="../share_reanalyses/" class="btn btn-xs btn-default pull-left">
        <i class="fa fa-chevron-left" aria-hidden="true"></i>
        Previous</a><a href="../share_reanalyses/" class="btn btn-xs btn-link">
        Share Reanalyses
      </a>
    </div>
    
  </div>

    

    <p>Start reanalyzing MassIVE datasets by clicking the “Reanalyze Spectra” button on the dataset page:</p>
<p><img alt="Image showing reanalyze button for MassIVE" src="../img/reanalyze_spectra/reanalyze_button.png" /></p>
<p>This brings you to the ProteoSAFe data analysis workflow input form, with the dataset’s spectrum files pre-selected. From here you can configure and run your search on these files:</p>
<p><img alt="Image showing complete reanalysis setup for MassIVE" src="../img/reanalyze_spectra/reanalysis_setup.png" /></p>
<p>ProteoSAFe offers a variety of analysis tools to derive meaningful knowledge from a dataset’s spectrum files:</p>
<table>
<thead>
<tr>
<th>Workflow</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr>
<td>Database Search</td>
<td><a href="http://proteomics.ucsd.edu/ProteoSAFe/index.jsp?params={%22workflow%22:%22MSGF_PLUS%22}">MS-GF+</a></td>
</tr>
<tr>
<td>Spectral Library Search</td>
<td><a href="http://proteomics.ucsd.edu/ProteoSAFe/index.jsp?params={%22workflow%22:%22MSPLIT_NEW%22}">MSPLIT</a></td>
</tr>
<tr>
<td>Multi-Blind Modification Search</td>
<td><a href="http://proteomics.ucsd.edu/ProteoSAFe/index.jsp?params={%22workflow%22:%22MODA%22}">MODa</a></td>
</tr>
<tr>
<td>De Novo Peptide Sequencing</td>
<td><a href="http://proteomics.ucsd.edu/ProteoSAFe/index.jsp?params={%22workflow%22:%22PEPNOVO%22}">PepNovo</a></td>
</tr>
<tr>
<td>Proteogenomics</td>
<td><a href="http://proteomics.ucsd.edu/ProteoSAFe/index.jsp?params={%22workflow%22:%22ENOSIC2%22}">Enosi</a></td>
</tr>
<tr>
<td>Multi-Pass Search</td>
<td><a href="http://proteomics.ucsd.edu/ProteoSAFe/index.jsp?params={%22workflow%22:%22MAESTRO%22}">Maestro</a></td>
</tr>
</tbody>
</table>

  <br>
    

    
    
      
    

  <div class="row wm-article-nav-buttons" role="navigation" aria-label="navigation">
    
    <div class="wm-article-nav pull-right">
      <a href="../access_public_datasets/" class="btn btn-xs btn-default pull-right">
        Next
        <i class="fa fa-chevron-right" aria-hidden="true"></i>
      </a>
      <a href="../access_public_datasets/" class="btn btn-xs btn-link">
        Access Public Datasets
      </a>
    </div>
    
    <div class="wm-article-nav">
      <a href="../share_reanalyses/" class="btn btn-xs btn-default pull-left">
        <i class="fa fa-chevron-left" aria-hidden="true"></i>
        Previous</a><a href="../share_reanalyses/" class="btn btn-xs btn-link">
        Share Reanalyses
      </a>
    </div>
    
  </div>

    <br>
</div>

<footer class="container-fluid wm-page-content">
      <p>
        <a href="https://github.com/CCMS-UCSD/MassIVEDocumentation/blob/master/docs/reanalyze_spectra.md">Edit on </a>
      </p>
  <p>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a> using <a href="https://github.com/gristlabs/mkdocs-windmill">Windmill</a> theme by Grist Labs.</p>
</footer>

</body>
</html>